print/ghostscript/Portfile

```diff
@@ -72,7 +72,6 @@ post-extract {
 
 patchfiles-append   patch-base_unix-dll.mak.diff
```

Comment:
FYI, I don't think you need this anymore. See https://bugs.ghostscript.com/show_bug.cgi?id=705907.

Reply:
The suggested patch from the developers did not fix the issue that the Macports patch `patch-base_unix-dll.mak.diff` is to fix. Maybe it served some other purpose.

I tried. When *not* applying `patch-base_unix-dll.mak.diff`, the resulting library (libgs.10.dylib) has a relative name, leading to the error message when completing the build:
```
Could not open libgs.10.dylib: Error opening or reading file (referenced from /opt/local/bin/gsc)
libgs.10.dylib seems to be referenced using a relative path. This may be a problem with its canonical library name and require the use of install_name_tool(1) to fix.
```
So we keep this patch and the post-patch replacement in place!

Reply:
Ah, the upstream fixes work for Homebrew because `brew` fixes up install names after the build. It would be worth filing another issue upstream to get this fixed.

Reply:
Oh, actually, I think you can use the fixes by upstream and drop the patch if you do something like
```sh
export DARWIN_LDFLAGS_SO_PREFIX="$PREFIX/lib/"
```
before calling `configure`, where `$PREFIX` is presumably `/opt/local`.

Reply:
Thanks @carlocab for that suggestion. Indeed removing the patch and setting the environment variable worked. That's now in the current PR.